roomify.install

```diff
@@ -318,8 +318,6 @@ function roomify_install_finished(&$install_state) {
   variable_set('addtoany_additional_html_placement', 'before');
   variable_set('addtoany_image', 'a2a_svg_32');
 
-  variable_set('commerce_checkout_completion_message', $message);
```

Comment:
do we set this on installation at all? I only see it being set in an update hook in roomify_system.

`roomify/roomify_accommodation_booking/roomify_accommodation_booking.admin.inc:  $form['roomify_booking_settings']['roomify_accommodation_booking_checkout_settings_fieldset']['commerce_checkout_completion_message'] = variable_form_element(variable_get_info('commerce_checkout_completion_message'));
roomify/roomify_accommodation_booking/roomify_accommodation_booking.install:  $variable_realm_list[] = 'commerce_checkout_completion_message';
roomify/roomify_accommodation_booking/roomify_accommodation_booking.install:  $variable_realm_list[] = 'commerce_checkout_completion_message';
roomify/roomify_accommodation_booking/roomify_accommodation_booking.variable.inc:  $variables['commerce_checkout_completion_message'] = array(
roomify/roomify_system/roomify_system.install:  variable_set('commerce_checkout_completion_message', $message);
`